fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/api/biome/v1/ModificationPhase.java

```diff
@@ -61,5 +62,14 @@ public enum ModificationPhase {
 	 * <p><b>Examples:</b> Mods that allow modpack authors to customize world generation, changing biome
 	 * properties (i.e. category) that other mods rely on.
 	 */
-	POST_PROCESSING
+	POST_PROCESSING;
+
+	public org.quiltmc.qsl.worldgen.biome.api.ModificationPhase asQuiltPhase() {
+		return switch (this) {
+		case ADDITIONS -> org.quiltmc.qsl.worldgen.biome.api.ModificationPhase.ADDITIONS;
+		case REMOVALS -> org.quiltmc.qsl.worldgen.biome.api.ModificationPhase.REMOVALS;
+		case REPLACEMENTS -> org.quiltmc.qsl.worldgen.biome.api.ModificationPhase.REPLACEMENTS;
+		case POST_PROCESSING -> org.quiltmc.qsl.worldgen.biome.api.ModificationPhase.POST_PROCESSING;
+		};
+	}
```

Comment:
It might be easier to do something like what [ResourcePackActivationType](https://github.com/QuiltMC/quilted-fabric-api/blob/HEAD/fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/api/resource/ResourcePackActivationType.java) does, but it's up to you really.